src/uwtools/rocoto.py

```diff
@@ -0,0 +1,44 @@
+"""
+Support for writing a Rocoto XML using J2Templating
+"""
+
+from uwtools.j2template import J2Template
+from uwtools.config import YAMLConfig
+
+# private functions
+def _add_jobname(tasks: dict) -> None:
+    """
+    Add the jobname entry for all the tasks in the workflow.
+
+    :param tasks: Dict of tasks in workflow.
+    """
+    if not isinstance(tasks, dict):
+        return
```

Comment:
These two lines are not needed because the `tasks` argument is type-annotated and we use typechecking. In an API method exposed to outside users we might want to do such a check.